app/src/main/java/org/oppia/android/app/settings/profile/ProfileResetPinActivityPresenter.kt

```diff
@@ -31,7 +31,7 @@ class ProfileResetPinActivityPresenter @Inject constructor(
   }
 
   fun handleOnCreate() {
-    activity.title = activity.getString(R.string.profile_reset_pin_title)
+    activity.title = activity.getString(R.string.profile_reset_pin_activity_title)
```

Comment:
@rt4914 is this still needed if it's being set in XML? It seems unnecessary. @4shutosh I think you can remove this.